cumulusci.yml

```diff
@@ -221,5 +221,7 @@ plans:
                         name: Course Connection Record Types for EDA
                     facility_display_name:
                         name: Facility Display Name Formula Field
+                    case_behavior_record_types:
```

Comment:
Super minor, but now that we've moved this to the post-install, it bothers me that "Course Connection Record Types for EDA" is inconsistent with our naming for all other record types...

Reply:
I'm okay with renaming it in this PR if there's consensus on that. Otherwise we can revisit this next week.

Reply:
Are we talking about changing it to "EDA - Course Connection Record Types" to make it consistent? If so, I'm in favor.